### ISSUE TITLE
Allow videos to be opened in a new tab

### ISSUE THREAD

**Issue by GlassedSilver** [Tzahi12345/YoutubeDL-Material#215]
Especially since I'm working on my deployment a lot as I may consume content I would love to right click and open in new tab a bunch of stuff, also for easier queuing of what to watch.

This is also one of the reasons why I often prefer watching YT on the PC over my apps, since queuing up stuff with tabs is much easier that way. :)

**Comment by Tzahi12345** [comment]
Surprised I hadn't thought of this! Adding a context menu on a right click is a bit overkill, but making it so ctrl-click opens the video in a new tab is trivial.

Well, decided to take a stab at it and I got it working! You can check it out in the latest `nightly` thx for the suggestions as always :)

[Relevant commit](https://github.com/Tzahi12345/YoutubeDL-Material/commit/35d0d439fadf318981f64d68b2470a851725af73)

**Comment by GlassedSilver** [comment]
The problem about this is that it doesn't work on mobile...

And why require two hands on PC? Most other sites let me midde-mouse-click something to open it in a new tab.

I wasn't asking for a custom context menu btw, just the regular things we're all used to? Hmm

**Comment by Tzahi12345** [comment]
That's a really good point, I didn't realize it wouldn't work on mobile.

The reason why it can't have the regular right click action menu is because it isn't a normal link, all the navigation logic is happening dynamically in TypeScript once a left click is detected.

I'll think of a good way to fix this issue, I'll reopen in the meantime. Good catch!

**Comment by Tzahi12345** [comment]
Adding a context menu was easier than I thought! I added a new menu with two options, either open the file in the player or open it in a new tab (see screenshot below). [Here's the relevant commit](https://github.com/Tzahi12345/YoutubeDL-Material/commit/3d2d4efb31b4161aebea02d3404d36c5f378ce97). Tested it out on mobile and it works with no issues on long press (at least on Android)

Check out the latest `nightly`, thanks for the suggestions as always :)

**Screenshot:**
![image](https://user-images.githubusercontent.com/707413/94332652-f0545700-ffa4-11ea-8252-224eaeb86094.png)
